docs/api/IoT.js-API-Buffer.md

```diff
@@ -21,26 +21,26 @@ The following shows Buffer module APIs available for each platform.
 
```

Comment:
I think it would be good to add a Legend for each doc, and select a more clear mark to indicate the implemented features. I've created an [example gist](https://gist.github.com/bzsolt/b8badcf29a7693d977e64c241aeb053f) with a few check-mark style.
(I prefer the first one :))